apstools/filewriters.py

```diff
@@ -657,7 +658,7 @@ def myPlan():
         #C Mon Jan 28 12:48:14 2019.  exit_status = success
 
     """
-    # global specwriter       # such as: specwriter = SpecWriterCallback()
+    global specwriter       # such as: specwriter = SpecWriterCallback()
```

Comment:
Codacy found an issue: [Using global for 'specwriter' but no assignment is done](https://app.codacy.com/gh/BCDA-APS/apstools/pullRequest?prid=5764884)